### PR TITLE
Feature/route overview info

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 1]
```diff
@@ -130,6 +132,38 @@
         }
 
         /**
+         * Get a summary for a route, including time, distance and # of turns
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js]: This data comes through on the `leg` object, including the total distance and start and end time, so it's not necessary to calculate it from the individual steps.

File: src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js [hunk 2]
```diff
@@ -123,6 +129,8 @@
                     resetStyleOnMouseout: true
                 }
             });
+            var walkSpeed = (currentUser.preferences.speed || 2) * 0.44704;
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/routing/overview/overview-controller.js]: Would be good to have a note here, or better, put this multiplier in a well-named constant.

File: src/nih_wayfinding/app/scripts/routing/directions-service.js [hunk 3]
```diff
@@ -130,6 +132,38 @@
...
+                return sum + add;
+            }, 0);
+            var distance = _.reduce(properties, function (sum, n) {
+                return sum + n.directions.distanceMeters;
+            }, 0);
+            walkSpeed = walkSpeed || 1;
+            return {
+                distance: distance, // m
+                turns: turns,
+                time: distance / walkSpeed / 60 // mins
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js]: We should be setting the walk speed with the OTP query parameter.

Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js]: Ah forgot about that. I'll make that change then.

File: src/nih_wayfinding/app/scripts/routing/directions-service.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/routing/directions-service.js]: Right. Although, the geojson format we set up doesn't really allow for a trip level properties object. Let me see if I can reshuffle things a bit, maybe save the itinerary summary to a private var in Directions.

**Comment by CloudNiner** [comment]
@flibbertigibbet PR updated

**Pull request closed without merging**